addons/account/models/ir_attachment.py

```diff
@@ -175,3 +175,10 @@ def action_download_xsd_files(self):
         # To be extended by localisations, where they can download their necessary XSD files
         # Note: they should always return super().action_download_xsd_files()
         return
+
+    def _post_add_create(self, **kwargs):
+        move_attachments = self.filtered(lambda attachment: attachment.res_model == 'account.move')
+        moves_per_id = self.env['account.move'].browse([record.res_id for record in move_attachments]).grouped('id')
+        for record in move_attachments:
+            moves_per_id[self.res_id]._check_and_decode_attachment(record)
```

Comment:
It will only work if `self` is a singleton
I guess flg meant:
```suggestion
        for attachment in move_attachments:
            moves_per_id[attachment.res_id]._check_and_decode_attachment(attachment)